tests/network-ovn

```diff
@@ -2019,6 +2019,94 @@ ovn_l3only_tests() {
     lxc network delete lxdbr0
 }
 
+# countDynamicLeases is a helper function that counts dynamic leases of the instance within
+# all network leases.
+countDynamicLeases(){
+        network="$1"
+        instance="$2"
+        count=$(lxc network list-leases "${network}" --format compact | awk -v inst="${instance}" '$1 == inst && $4 == "DYNAMIC" { count++ } END { print count+0 }')
+        echo "${count}"
+}
+
+# countStaticLeases is a helper function that counts static leases of the instance within
+# all network leases.
+countStaticLeases(){
+        network="$1"
+        instance="$2"
+        count=$(lxc network list-leases "${network}" --format compact | awk -v inst="${instance}" '$1 == inst && $4 == "STATIC" { count++ } END { print count+0 }')
+        echo "${count}"
+}
+
+# hasLeaseIP is a helper function that checks whether the instance has the given static lease IP.
+hasLeaseIP(){
+        network="$1"
+        instance="$2"
+        ip="$3"
+        result=$(lxc network list-leases "${network}" --format compact | awk -v inst="${instance}" -v ip="${ip}" '$1 == inst && $3 == ip { print $3 }')
+        [ -z "${result}" ] && echo 1 || echo 0
```

Comment:
IMHO, `return`'ing 1 or 0 instead of echo'ing would make that function easier to use:

```
# Check if the instance got an IP
hasLeaseIP test-ovn c1 "10.10.11.15"

# Ensure the instance doesn't that have an IP
! hasLeaseIP test-ovn c1 "10.10.11.15" || false
```